main.js

```diff
@@ -30,6 +30,28 @@ function disableAnimationsForTest() {
     }
 }
 
+function setupInspectMenu() {
```

Comment:
@kelsos What is this function for?

Reply:
This adds the `inspect` action on the context menu for development, but without having to depend on third-party libraries. I already checked that this is not enabled for the packaged build, but I would like if you could also verify it just to be on the safe side.

Reply:
But I just checked and this seems to be always on. Can you make it activated only by a switch or something? People can use rotkehlchen from source and does not mean they want to have development tools on by default.

Reply:
You mean by using `npm start`, yes this was intentional. I was referring to the packaged build. I can change it, though I cannot imagine why would anyone go through the trouble of installing node.js and all the dependencies other than coding :P

Reply:
There are people like me, Rakan and probably you who use the program by source. So the end result should by default be like what the final binary should be.

Reply:
Just make this a switch we can activate when developing to enable the right click context menu. @kelsos ... do you need this in the automatic UI tests?

Reply:
I need it to build the tests mostly